packages/edit-post/src/components/layout/index.js

```diff
@@ -100,7 +96,6 @@ function Layout() {
 		editorSidebarOpened || pluginSidebarOpened || publishSidebarOpened;
 	const className = classnames( 'edit-post-layout', 'is-mode-' + mode, {
 		'is-sidebar-opened': sidebarIsOpened,
-		'has-fixed-toolbar': hasFixedToolbar,
```

Comment:
Is this something that you think is worth mentioning on a dev note?

Reply:
Hm, I saw this as largely a clean-up of classes which were no longer used in the editor, but never removed. To answer your question, I guess it depends on a combination of (a) are plugins using it and (b) are there use-cases for it? I don't have a huge concern of reverting this specific change, aside from the fact that it becomes dead code (maintenance burden, larger bundle sizes).

I do see a couple results here: https://wpdirectory.net/search/01E3J9B3K378P5PV2104804D06

But it's hard to tell if the usage is intended for `has-fixed-toolbar` of the layout component, vs. `has-fixed-toolbar` of the _block_.